kubernetes/zuul/components/zuul-executor/statefulset.yaml

```diff
@@ -58,19 +58,36 @@ spec:
             - containerPort: 7900
               name: "logs"
               protocol: "TCP"
+            - containerPort: 9091
+              name: "prometheus"
+              protocol: "TCP"
+
+                #          readinessProbe:
```

Comment:
are probes not working properly?

Reply:
the executor one is somehow tricky. Old style we had no probes for executor either